src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -823,12 +823,12 @@ private static void createAndCheckIsDirectory(Path dir,
      * names in the same manner as the {@link
      * java.io.File#createTempFile(String,String,File)} method.
      *
-     * <p> As with the {@code File.createTempFile} methods, this method is only
-     * part of a temporary-file facility. Where used as a <em>work files</em>,
-     * the resulting file may be opened using the {@link
+     * <p> The resulting file should be deleted once it is no longer needed.
```

Comment:
I think this sentence can be omitted as the sentence which follow the above I believe can stand on its own

Reply:
I concur. Fixed.